workshops/util.py

```diff
@@ -431,18 +431,26 @@ def parse_tags_from_event_website(tags):
     try:
         latitude, _ = tags.get('latlng', '').split(',')
         latitude = float(latitude.strip())
-    except ValueError:
+    except (ValueError, TypeError, AttributeError):
+        # value error: can't convert string to float
+        # type error: can't convert None to float
```

Comment:
How do you get that far?  If `tags.get(…)` gives you `None`, the `split(…)` will raise an `AttributeError`.  If `tags.get(…)` doesn't give you `None`, I don't see how you'd get a `None` later.

Also, I don't think you need the `strip()`, since:

    >>> float(' 123 ')
    123.0

but that's not code you're touching in this PR, and it shouldn't hurt anything to have the `strip()` call.